ingest/Snakefile

```diff
@@ -8,6 +8,11 @@ workdir: workflow.current_basedir
 # Use default configuration values. Override with Snakemake's --configfile/--config options.
 configfile: "defaults/config.yaml"
 
+segments = ['L', 'S']
```

Comment:
[using this line just to organise discussion into a thread]

The [2018 Nextstrain Lassa builds](https://next.nextstrain.org/staging/lassa/S:staging/lassa/L) have the majority of the strains matched up. Was this using a subset of the data? Or were we perhaps not considering strain name duplications and so the matching may be incorrect? 

Matching samples across segments is a key part of segmented analyses. Absolutely fine to not be part of this PR, and perhaps is something we can never generalise, but is something we should certainly keep revisiting.

Reply:
> The [2018 Nextstrain Lassa builds](https://next.nextstrain.org/staging/lassa/S:staging/lassa/L) have the majority of the strains matched up. Was this using a subset of the data? Or were we perhaps not considering strain name duplications and so the matching may be incorrect?

Good question. I have no idea how the underlying data for those builds were processed. The [early README](https://github.com/nextstrain/lassa/blob/40b1f01db05b364ce6f283f56dc77787bf106dfe/README.md#fauna--rethinkdb-credentials) implies the data came from fauna, but I don't see any lassa table in fauna...

> Matching samples across segments is a key part of segmented analyses. Absolutely fine to not be part of this PR, and perhaps is something we can never generalise, but is something we should certainly keep revisiting.

For sure, feel free to add thoughts on segment analysis to https://github.com/nextstrain/pathogen-repo-guide/issues/59.